Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.yml

```diff
@@ -100,6 +100,12 @@ configuration:
   type: 8
   additionalinfo: 'A parameter integration that removes the playbook ID field from incoming incidents. Note: When set to true (default), the instance will attempt to run a playbook according to the incoming ID. When set to false, the instance will run the default playbook for the incident type (if configured locally).'
   required: false
+- display: Reset mirror 
+  name: reset_mirror
+  defaultvalue: "false"
+  type: 8
+  additionalinfo: 'Will mirror all notes,tags and attachments for incidents. Notice can impact performance if combined with "Reset the "last run" timestamp" and multiple incidents in system.'
```

Comment:
This description should reference fetching the history rather than "all" - as it's relevant just for the first mirror.